### PR TITLE
Updated trigger changelog script

### PR DESCRIPTION
**Pull request by BenStarmerSmith**
## Proposed changes

Updated the documentation for the trigger changelog script. Fixed how the web addresses are linked.

## Checklist

Before posting a PR ensure that from each of the below categories **AT LEAST ONE BOX HAS BEEN CHECKED**. If more than one category is applicable then more can be checked. Also ensure that the proposed changes have been filled out with relevant information for reviewers.

## Tests

- [ ] No tests required to be added. (For small changes that will be tested by CI/CD infrastructure).
- [ ] Added/Modified automated and PhantomHand CI tests (if a new class is added (Python or C++), the interface of that class must be unit tested).
- [ ] Manually tested in simulation (if simulation specific or no hardware required to test the functionality). 
- [x] Manually tested on hardware (if hardware specific or related).

## Documentation

- [ ] No documentation required to be added.
- [ ] Added documentation (For any new feature, explain what it does and how to use it. Write the documentation in a relevant space, e.g. Github, Confluence, etc).
- [x] Updated documentation (For changes to pre-existing features mentioned in the documentation).

### CODE REVIEW

**Review by maxzieba — COMMENTED** [review]
Can you show the output difference?

File: bin/README.md [hunk 1]
```diff
@@ -113,14 +113,14 @@ This script is used to allow you to pull a changelog of a specific image between
 
 ## Usage
 To pull the changelog between two dates run the script like this:</br>
-```python3 trigger_changelog.py -u <github_username> -t <github_token> -ir <image_repository> -ib <image_branch (default: noetic-devel)> -sd <start_date> -ed <end_date>```</br>
```
Comment (maxzieba) [inline bin/README.md:116]: What did 'i' do in the first place?

**Review by maxzieba — APPROVED** [review]

File: bin/README.md (hunk 1 above)
Comment (bdenoun) [reply bin/README.md:116]: Extracted from `python3 --help`:

> -i     : inspect interactively after running script; forces a prompt even
         if stdin does not appear to be a terminal; also PYTHONINSPECT=x

**Review by bdenoun — APPROVED** [review]

File: bin/README.md (hunk 1 above)
Comment (BenStarmerSmith) [reply bin/README.md:116]: i had -ib and -ir to stand for image branch and image repo, but it didn't make sense

**Comment by BenStarmerSmith** [comment]
Output difference:
Forcing bag_rotate to only handle files with specific prefix  ([Github|(https://github.com/shadow-robot/common_resources/pull/235])
vs
Updating new calibration gui to prevent old gui breaking - https://github.com/shadow-robot/sr_hand_glove_internal/pull/22
I had it preformatted like this to try making it easier to link in jira. But it didn't work

**Pull request merged**